src/syntax/constraint_helpers.ts

```diff
@@ -27,6 +27,18 @@ const taggedConstraint = (key: string | number | symbol) => (value: any) => {
     return constraint;
 };
 
+const multiTaggedConstraint = (...tags: [interfaces.Tag, ...interfaces.Tag[]]) => {
+    const constraint: interfaces.ConstraintFunction = (request: interfaces.Request) =>
```

Comment:
`Request | null`

Reply:
And full coverage

Reply:
There are some uncovered lines in Container.resolve that are mine
https://github.com/inversify/InversifyJS/pull/1347